cli/kafka-consumer-groups.md

```diff
@@ -0,0 +1,19 @@
+# kafka-consumer-groups.sh examples
+
+Examples of using the `kafka-consumer-groups.sh` provided in kafka bin directory.
+
+## Describe a consumer group, see the topics (and its partitions) it is consuming and each partition's the current offset (already consumed by the consumer group).
```

Comment:
What about create a header called "Describe Consumer Group", and move the current sentence as paragraph?

Reply:
Fixed